src/components/motorcycleForm/addMotorcycle.js

```diff
@@ -0,0 +1,121 @@
+import { useState, useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { createMotorcycle } from '../../redux/motorcycles/motorcycles';
+
+const AddMotorcycle = () => {
+  const getData = () => ({
+    model: '',
+    price: 0,
+    duration_months: 0,
+    description: '',
+    image: '',
+  });
```

Comment:
I suggest you set `getData` as an object directly rather than a function that returns an object.
For example
```
const getData = {
  model: '',
  price: 0,
  duration_months: 0,
  description: '',
  image: '',
 };
```